draft-packages/form/KaizenDraft/Form/TextAreaField/README.mdx

```diff
@@ -0,0 +1,151 @@
+demoStoryId: textareafield-react--default-kaizen-site-demo
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+import DoAndDontContainer from "docs-components/DoAndDontContainer"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D50186%253A40775&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## Copy guidelines
+
+To be scannable and fully helpful, a text area must have an associated label. Here’s what you should know:
+
+### Give it a clear, short label
+
+Make sure your text area has a short, descriptive label above it that describes what the user should type into the box below. Avoid phrasing your labels as questions. Keep in mind that your label shouldn’t contain instructions. Reserve those for the helper text.
+
+<DoAndDontContainer>
+<Do>
+
+- Comment
+
+</Do>
+
+<Dont>
+
+- Enter your comment here
+
+</Dont>
+</DoAndDontContainer>
+
+### Placeholder text should be examples and not instructions
+
+Make sure that any text inside the field box serves as an example of what the user should type into the box. Don’t put any instructions inside the box. 
+
+<DoAndDontContainer>
+<Do>
+
+- e.g. Thanks for everything!
+
+</Do>
+
+<Dont>
+
+- Thanks for everything!
+
+</Dont>
+</DoAndDontContainer>
+
+Keep in mind that examples aren’t always required. For example, a text area asking for someone’s opinion doesn’t need an example in it if it will be misleading. Only use placeholder text when required. More often than not, blank text fields are easier to spot and scan.
+
+### Error text should be succinct
+
+Keep all error text short, useful and to the point.
+
+<DoAndDontContainer>
+<Do>
+
+- You must enter a comment
+
+</Do>
+
+<Dont>
+
+- You entered a comment that is invalid
+
+</Dont>
+</DoAndDontContainer>
+
+### Helper text should provide extra guidance
+
+Helper text placed below the field can include further instructions or information on how to fill out the text area. It can also include details on what will happen with the filled out information. Don’t put placeholders here.
+
+<DoAndDontContainer>
+<Do>
+
+- Your comment will be used anonymously
+
+</Do>
+
+<Dont>
+
+- For example, “Thanks for everything!”
+
+</Dont>
+</DoAndDontContainer>
+
+### Write in sentence case with minimal punctuation
+
+All text area copy should be in sentence case i.e. the first letter of the first word is capitalized and everything else is in lower case unless it’s a proper noun or feature name. Always err on the side of minimal punctuation, so leave out full stops and commas where possible.
+
```

Comment:
Might we have a section here on resizing i.e. fixed containers vs resizable (when and where), do we forbid horizontal resizing etc.